src/styles/product-card-styles/card.scss

```diff
@@ -0,0 +1,104 @@
+@import "variables.scss";
+
+.card {
+  margin: 0;
+  padding: 32px 16px 0;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &:hover {
+    transition: all 300ms;
```

Comment:
Transition should be added for the default state of the element, not hover. In that case, everything will work smoothly
![transition](https://github.com/mate-academy/layout_catalog/assets/96409563/11a36f79-908c-470d-9a76-16bd0674ec32)